roles/lnls-ans-role-desktop-apps/tasks/setup-Debian.yml

```diff
@@ -15,6 +15,8 @@
     state: "{{ desktop_packages_state }}"
     install_recommends: "{{ desktop_install_recommends }}"
     default_release: "{{ desktop_default_release | default(omit) }}"
+    allow_unauthenticated: true
+    force: true
```

Comment:
not sure how dangerous this is. I have to use it do down downgrade `ca-certificates` repos in con newly created debian-9.12 host.

Reply:
@lerwys  pointed out frightening warnings on the use of this `force: true` option. I am removing this option...